src/gpu/RaytraceRequestContext.hpp

```diff
@@ -20,16 +20,22 @@
 struct RaytraceRequestContext
 {
 	// Input
+	Vec6f* sensorVelocity;
+
```

Comment:
1. Why Vec6f? IMHO it is less obvious than 2xVec3f.
2. I'd use pass-by-value, same as rayOriginToWorld is passed.
```suggestion
	Vec3f sensorLinearVelocity;
	Vec3f sensorAngularVelocity;
```